assets/components/templates/base/base.twig

```diff
@@ -8,16 +8,16 @@
 {% endblock %}
 
 <div class="main-container">
-  {% if not isHomepage %}
   {% block breadcrumb %}
+    {% if not isSpecial %}
```

Comment:
Not super convinced about the `isSpecial` name 😅 It sounds a bit like `isTheChoosenOne`. I agreed that `isHomepage` became quickly out of strict homepage usage, but I think `hasSidebar` or `noSidebar` (if you prefer not refactoring all the logic) can be a better way to do it 😉

Reply:
well... we discussed this issue with @nsz and came with this super name 🥇 

isNotStandard meaning itDoesntHaveMenuBreadcrumbAndNavAside
Can we just say that if we need a second layout child type we refactor the logic ?